package-manifest.toml

```diff
@@ -15,6 +16,7 @@ source.type = "local"
 source.rust.binary_names = ["crucible-pantry"]
 source.rust.release = true
 source.paths = [
-  { from = "pantry/smf", to = "/var/svc/manifest/site/crucible_pantry" }
+  { from = "pantry/smf", to = "/var/svc/manifest/site/crucible_pantry" },
```

Comment:
Let's just make this path `/var/svc/manifest/site/crucible` like the other one, and renaming `manifest.xml` to `pantry.xml` will mean the full path will end up being `/var/svc/manifest/site/crucible/pantry.xml`.

Reply:
Done